_1327/documents/tests.py

```diff
@@ -201,3 +203,187 @@ def test_view_permissions(self):
 
 			permission_names = [permission[0] for permission in subclass._meta.permissions]
 			self.assertIn(subclass.VIEW_PERMISSION_NAME, permission_names, msg="All Document subclasses must declare the permission named in VIEW_PERMISSION_NAME")
+
+
+class TestAttachments(WebTest):
+	"""
+		Tests that test the work with attachment.
+		Informationdocuments are used as testclass. It is assumed that the behaviour is similar with other documenttypes
```

Comment:
Please use correct capitalization for classes, "behavior", and change the first sentence to "Tests creating, viewing and deleting attachments".